scripts/ganache/makeSnapshot.js

```diff
@@ -0,0 +1,19 @@
+const makeSnapshot = () => {
+  return new Promise((resolve, reject) => {
+    web3.currentProvider.send({
```

Comment:
We have a nice async wrapper around this send method [here](https://github.com/gnosis/dex-contracts/blob/master/test/utilities.js#L105), which would be more concise to use.

Reply:
Do you mean something like `await web3.currentProvider.send(...)`

Reply:
Something like https://github.com/gnosis/dex-contracts/blob/master/test/utilities.js#L119

Reply:
Yes, thanks! I will see how that works!